### PR TITLE
Fix part of #1287: Fix lint issues in app module in the kotlin files (conceptcard, info, questionplayer)

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fixes part of #1287
`conceptcard, info, questionplayer` folder `ktlint` check

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by vinitamurthi — APPROVED** [review]

File: app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardFragment.kt [hunk 1]
```diff
@@ -29,7 +29,8 @@ class ConceptCardFragment : InjectableDialogFragment() {
     }
   }
 
-  @Inject lateinit var conceptCardFragmentPresenter: ConceptCardFragmentPresenter
+  @Inject
```
Comment (nikitamarysolomanpvt) [inline app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardFragment.kt:32]: Please revert this change to
 `@Inject lateinit var conceptCardFragmentPresenter: ConceptCardFragmentPresenter`

File: app/src/main/java/org/oppia/app/topic/info/TopicInfoFragment.kt [hunk 2]
```diff
@@ -24,14 +24,19 @@ class TopicInfoFragment : InjectableFragment() {
     }
   }
 
-  @Inject lateinit var topicInfoFragmentPresenter: TopicInfoFragmentPresenter
+  @Inject
```
Comment (nikitamarysolomanpvt) [inline app/src/main/java/org/oppia/app/topic/info/TopicInfoFragment.kt:27]: ditto.

File: app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerActivity.kt [hunk 3]
```diff
@@ -10,14 +10,19 @@ import org.oppia.app.player.stopplaying.StopExplorationDialogFragment
...
-class QuestionPlayerActivity : InjectableAppCompatActivity(), StopStatePlayingSessionListener,
-  RestartPlayingSessionListener, StateKeyboardButtonListener {
+class QuestionPlayerActivity :
+  InjectableAppCompatActivity(),
+  StopStatePlayingSessionListener,
+  RestartPlayingSessionListener,
+  StateKeyboardButtonListener {
 
-  @Inject lateinit var questionPlayerActivityPresenter: QuestionPlayerActivityPresenter
+  @Inject
```
Comment (nikitamarysolomanpvt) [inline app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerActivity.kt:24]: ditto.

File: app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerFragment.kt [hunk 4]
```diff
@@ -18,19 +18,30 @@ import org.oppia.app.player.state.listener.SubmitNavigationButtonListener
...
+  InteractionAnswerErrorReceiver,
+  ContinueNavigationButtonListener,
+  NextNavigationButtonListener,
+  ReplayButtonListener,
+  ReturnToTopicNavigationButtonListener,
+  SubmitNavigationButtonListener,
   PreviousResponsesHeaderClickListener {
 
-  @Inject lateinit var questionPlayerFragmentPresenter: QuestionPlayerFragmentPresenter
+  @Inject
```
Comment (nikitamarysolomanpvt) [inline app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerFragment.kt:32]: ditto.

**Review by nikitamarysolomanpvt — CHANGES_REQUESTED** [review]
PTAL on the nit changes

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM. I resolved all `@Inject` alignment related comments as it is advised to do so.

**Comment by anandwana001** [comment]
> LGTM. I resolved all `@Inject` alignment related comments as it is advised to do so.

Thank You :)

**Pull request merged**